src/events/render-advert.ts

```diff
@@ -200,6 +201,26 @@ const renderAdvert = (
 	advert: Advert,
 	slotRenderEndedEvent: googletag.events.SlotRenderEndedEvent,
 ): Promise<boolean> => {
+	const isA9GumGum =
+		(window.guardian.a9WinningBids?.[0]?.['amznp'] ?? '') === '1lsxjb4';
```

Comment:
You need find the bid response for this specific ad slot, `0` may not necessarily be this slot

Reply:
I was looking at the email from APS where they state:

"This amznp value denotes which bidder this bid is from, and you can use your Bidder Mapping sheet to see which specific bidder this maps to (this is available under Demand > Manage Demand in your UI)."

So from that I infered it was the winning bid, therefore I thought the 0'th being the only response in the array?
![image](https://github.com/user-attachments/assets/22a705bc-fa4e-4cc0-87f3-cfc828d1f699)

@Jakeii

Reply:
right, but fetch bids requests ads for multiple slots (in our case, not in the screenshot)

Reply:
I see we are getting an array of bidResponse objects, I have dealt with this by matching each 'advert` being processed by the `render-advert` function to be matched to the array of bidResponses.